Tests/QuranAudioKitTests/Sura+Fixture.swift

```diff
@@ -0,0 +1,19 @@
+//
+//  File.swift
+//  
+//
+//  Created by Mohammad Abdurraafay on 2023-02-11.
+//
+
+import Foundation
+import QuranKit
+
+extension Sura {
+    static var firstSura: Sura {
+        Quran.madani.firstSura
+    }
+    
+    static var secondSura: Sura {
+        Quran.madani.suras[1]
+    }
+}
```

Comment:
Let's not do this because soon I'll introduce other Quran reading types and Sura.firstSura wouldn't make sense. Instead, let's be a little bit verbose and use `Quran.madani.firstSura`.